src/main/scala/com/snowflake/snowpark/Row.scala

```diff
@@ -367,6 +367,39 @@ class Row protected (values: Array[Any]) extends Serializable {
     getAs[Map[T, U]](index)
   }
 
+  /**
+   * Returns the value at the specified column index and casts it to the desired type `T`.
+   *
+   * Example:
+   * {{{
+   *   val row = Row(1, "Alice", 95.5)
+   *   row.getAs[Int](0) // Returns 1 as an Int
+   *   row.getAs[String](1) // Returns "Alice" as a String
+   *   row.getAs[Double](2) // Returns 95.5 as a Double
+   * }}}
+   *
+   * @param index the zero-based column index within the row.
+   * @tparam T the expected type of the value at the specified column index.
+   * @return the value at the specified column index cast to type `T`.
+   * @throws ClassCastException if the value at the given index cannot be cast to type `T`.
+   * @throws ArrayIndexOutOfBoundsException if the column index is out of bounds.
+   * @group getter
+   * @since 1.14.0
```

Comment:
please change this to `1.15.0`

Reply:
Fixed on commit [c8ebed8](https://github.com/snowflakedb/snowpark-java-scala/pull/148/commits/c8ebed8047687ea5de907a3cf9bc5c855b7b76b8)